code/game/machinery/Sleeper.dm

```diff
@@ -279,3 +279,25 @@
 
 /obj/machinery/sleeper/old
 	icon_state = "oldpod"
+
+/obj/machinery/sleeper/party
+	name = "party pod"
+	desc = "Sleepers were made illegal in the year 2559, as a result of finding doses of lead acetate in every chemical dosage. This is one of the left-overs, that's been retrofitted to a 'party pod'."

```

Comment:
something about this sentence is horribly structured and it's epicenter is at the first comma

Reply:
also what divine spirit told you to put a dash between left and overs

Reply:
also also doses of thing in chemical doses is just bad writing

Reply:
Brain no workie too good when nightcode

Reply:
I'm crashing this review with no survivors.

"doses of lead acetate in every chemical dosage"
replace dosage with something like injection, patient, ect.
"It's been retrofitted to a party pod."
replace to with Into/to be, or just fuck it and go with it's been recreated as a party pod.